packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/extensions/error_model_translations.dart

```diff
@@ -28,6 +28,14 @@ extension ErrorModelL10n on ErrorModel {
       return (this as ErrorServerGenericModel).translate(context);
     }
 
+    if (this is NoMailAppErrorModel) {
+      return (this as NoMailAppErrorModel).translate(context);
+    }
+
+    if (this is InvalidUrlErrorModel) {
+      return (this as InvalidUrlErrorModel).translate(context);
+    }
+
```

Comment:
I believe this should be conditionally added for when the relevant features are enabled.

Reply:
Done